.travis.yml

```diff
@@ -5,19 +5,14 @@ env:
   - NUMPY_VERSION=stable
   - MAIN_CMD='python setup.py'
   - CONDA_DEPENDENCIES='xarray dask toolz Cython pykdtree sphinx cartopy rasterio pillow matplotlib
-    pyyaml pyproj coveralls configobj coverage codecov'
+    pyyaml pyproj coveralls configobj coverage codecov pytest pytest-cov'
   - SETUP_XVFB=False
   - EVENT_TYPE='push pull_request'
   - SETUP_CMD='test'
   - CONDA_CHANNELS='conda-forge'
   - CONDA_CHANNEL_PRIORITY='strict'
 matrix:
   include:
-  - env: PYTHON_VERSION=2.7
-    os: linux
-  - env: PYTHON_VERSION=2.7
-    os: osx
-    language: generic
```

Comment:
Should we add python 3.8 instead ?

Reply:
I don't think we need to. conda-forge still isn't technically completely migrated to python 3.8.